coral/src/app/features/topics/acl-request/fields/AclTypeField.tsx

```diff
@@ -0,0 +1,27 @@
+import {
+  RadioButton as BaseRadioButton,
+  RadioButtonGroup as BaseRadioButtonGroup,
+} from "@aivenio/aquarium";
+
+interface AclTypeFieldProps {
+  handleChange: (value: string) => void;
+  topicType: string;
```

Comment:
I think I tried it at first but then ran into some complications. Checking it out quickly, I think the block is with the `onChange` prop of `BaseRadioButtonGroup`, which needs to be typed as the union type, but cannot be because it is `string`.

![Screenshot 2023-01-23 at 14 09 16](https://user-images.githubusercontent.com/20607294/214048816-5f387a22-6280-4da7-85f4-6bd6f5d14d2a.png)


We could cast it like this, but it feels like an annoying bug in waiting, even if it seems very safe at the moment.

![Screenshot 2023-01-23 at 14 15 19](https://user-images.githubusercontent.com/20607294/214048943-500049d2-dd83-4836-9e07-c98b190daf94.png)

Reply:
Yeah... it probably introduces more type safety related code than it yields value 🤔 . Lets keep it as is.